lib/transport/selenium-webdriver/options.js

```diff
@@ -328,7 +328,7 @@ module.exports = class SeleniumCapabilities {
 
   addHeadlessOption({options}) {
     if (this.argv.headless && (options instanceof Capabilities) && (this.isChrome || options.headless)) {
-      this.isChrome ? options.addArguments('headless=chrome') : options.headless();
+      this.isChrome ? options.addArguments('headless=new') : options.headless();
```

Comment:
What about the older versions of Chrome browser? Would `--headless=new` work with them as well?

Reply:
There is no way to detect the version of the chrome browser before the driver creation. And people using older version of chrome will likely be using older version of nightwatch

Reply:
We should tagged this as breaking change

Reply:
But we can also detect the version of `chromedriver`, right? Since chromedriver version == chrome version.
(Of course, only if it is worth doing. But I guess we can also pass the `--headless=chrome` argument directly in desired capabilities and it will work for older chrome versions?)